pkg/network/ebpf/c/protocols/postgres/types.h

```diff
@@ -7,10 +7,13 @@
 #define POSTGRES_BUFFER_SIZE 160
 
 // Represents the maximum number of tail calls we can use to process a single message.
-#define POSTGRES_MAX_TAIL_CALLS_FOR_MAX_MESSAGES 1
+#define POSTGRES_MAX_TAIL_CALLS_FOR_MAX_MESSAGES 3
```

Comment:
this constant is no longer used separately, may be can remove it and use POSTGRES_MAX_TOTAL_MESSAGES if needed?